src/tests/unit/pagination.test.ts

```diff
@@ -63,7 +63,7 @@ describe("cursor paginator", () => {
       { owner: ownerB, foo: "a", timestamp: "2021-04-01" },
       { owner: ownerA, foo: "b", timestamp: "2021-03-01" },
       { owner: ownerB, foo: "c", timestamp: "2021-02-01" },
-      { owner: ownerA, foo: "d", timestamp: "2021-01-01" },
+      { owner: ownerA, foo: "d", timestamp: "2021-01-01", deletedAt: "2002-12-01", },
```

Comment:
There is a test below which ensures ordering by a nullable field works correctly, and I thought it would be nice to ensure that the ordering with date types worked as well in that test. Thus I added a deletedAt date to foo "d" that is earlier then the deletedAt on foo "e" to ensure the sorted by DESC deletedAt works within both coalesced null fields and non null fields